src/transformers/testing_utils.py

```diff
@@ -533,3 +533,71 @@ def test_something():
         use_tf = os.getenv("USE_TF", False)
     """
     return unittest.mock.patch.dict(os.environ, kwargs)
+
+
+# the following code deals with async io between processes
+
+# adapted from https://stackoverflow.com/a/59041913/9201239
+import asyncio  # noqa
+
+
+class RunOutput:
```

Comment:
maybe `ProcessOutputCollector` if it's in src/ it needs to be named more specifically I think.

Reply:
it's not used outside of the wrapper - a private/internal class

Reply:
as test_utils is growing this may end up in its own module